packages/runtime/test/transport/files.test.ts

```diff
@@ -274,6 +278,24 @@ describe("Files query", () => {
 
         await conditions.executeTests((c, q) => c.files.getFiles({ query: q, ownerRestriction: OwnerRestriction.Peer }));
     });
+
+    test("files can be queried by wasViewedAt", async () => {
+        const file = await uploadFile(transportServices1);
+
+        const viewedFilesBeforeViewing = await transportServices1.files.getFiles({ query: { wasViewed: "" } });
```

Comment:
More self-explanatory would be:

```suggestion
        const viewedFilesBeforeViewing = await transportServices1.files.getFiles({ query: { wasViewed: "true" } });
```